packages/drivers/odsp-driver/src/fetchSnapshot.ts

```diff
@@ -236,6 +238,7 @@ async function fetchLatestSnapshotCore(
                         fetchTimeout = undefined;
                     }
                 });
+                const networkTime2 = performance.now() - start;
```

Comment:
networkTime2 is hard to associate to anything. Perhaps rename to  networkWithinJS or something similar ?

Reply:
How about networkTime2Browser & networkTimeJS ? see last commit